MapboxGeocoder.xcodeproj/xcshareddata/xcschemes/MapboxGeocoder watchOS.xcscheme

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Scheme
-   LastUpgradeVersion = "1340"
+   LastUpgradeVersion = "1400"
```

Comment:
#219 would consolidate these schemes and targets to make future upgrades cleaner.